src/bessel.jl

```diff
@@ -116,7 +116,7 @@ for afn in (:airyai, :airyaiprime, :airybi, :airybiprime,
         $afn(z::Complex64) = Complex64($afn(Complex128(z)))
     end
     if afn in (:airyaix, :airyaiprimex)
-        @eval $afn(x::Real) = x < 0 ? throw(DomainErrorNoArgs) : real($afn(complex(float(x))))
+        @eval $afn(x::Real) = x < 0 ? throw(DomainError(x, "`x` cannot be negative.")) : real($afn(complex(float(x))))
```

Comment:
I think rather than referring to the function arguments by name, we should instead say things like "input cannot be negative." It's not clear from the user's perspective what `x` is since they didn't have to write `x=`.